plugins/dell-dock/fu-plugin-dell-dock.c

```diff
@@ -240,20 +215,3 @@ fu_plugin_composite_cleanup (FuPlugin *plugin,
 
 	return fu_dell_dock_ec_reboot_dock (parent, error);
 }
-
-gboolean
-fu_plugin_activate (FuPlugin *plugin, FuDevice *device, GError **error)
-{
-	g_autoptr(FuDeviceLocker) locker = NULL;
-	if (!FU_IS_DELL_DOCK_EC (device)) {
```

Comment:
I guess this is lost, but not super important.

Reply:
I checked, the EC is the only one implementing the activate vfunc.